js/src/name/resolve.js

```diff
@@ -125,7 +123,7 @@ module.exports = (createCommon, options) => {
             ipfs.name.resolve(keyId, resolveOptions, (err, res) => {
               expect(err).to.not.exist()
               expect(res).to.exist()
-              expect(res.path).to.equal(`/ipfs/${value}`)
+              expect(res.path || res).to.equal(`/ipfs/${value}`) // TODO
```

Comment:
This seems wrong, tests should test one thing and here is a conditional. Also the `TODO` seems to be missing a comment about what/why

Reply:
yes! That will be changed 🙂

But for explaining the purpose, I am experiencing different responses when the daemon is online and offline. One has the path property, while the other does not have. I will keep this in `WIP` before solving this